src/data/injections.js

```diff
@@ -467,6 +467,20 @@ const AVAILABLE_INJECTIONS = [
       ],
     },
   },
+  {
+    id: "bug1629926",
+    platform: "desktop",
+    domain: "hangouts mic",
+    bug: "1629926",
+    contentScripts: {
+      matches: ["https://mail.google.com/*"],
```

Comment:
Does the issue only happen in Gmail, or does this need to apply on hangouts.google.com? (I'm not sure...)

Reply:
Yeah, the problem is only when calling from within the gmail interface. Calling works when calling on hangouts.google.com directly

Reply:
Sweet, thanks!